### ISSUE TITLE
Fails to build in windows 7

### ISSUE THREAD

**Issue by blacknapkins** [bokeh/bokeh#6244]
Hi,

I have been trying to build Bokeh on Windows 7 using python 3.5.3. 

With a fix to _version.py, which is implemented later on, I can build 0.12.4. This displays the simple line example as expected.

When I try to build with 0.12.5rc1 the line example does not display as expected but raises an exception as outlined below. All the later builds I have tested do not resolve this.

Chrome

![chrome](https://cloud.githubusercontent.com/assets/14756121/25760721/01e84518-31d0-11e7-85d8-d7e616eeb1ae.PNG)
 
IE 11

![ie11](https://cloud.githubusercontent.com/assets/14756121/25760740/17570bc8-31d0-11e7-91ea-e8e3dad2f988.PNG)

**Comment by mattpap** [comment]
This is fixed as a side effect in PR #6233.

**Comment by mattpap** [comment]
Even with this fix, there are some other tests failures:

1. client: unable to connect to server. I suspect windows' firewall has something to do with this.
2. datarange1d: numerical error. Usage of deep instead of approximate equality to compare floats.

**Comment by blacknapkins** [comment]
I have built branch mattpap/4153-signaling as identified in PR #6233  above on Windows 7.

The error with Chrome above does not show and the Simple Line Example displays as expected.

On IE 11 I get the stack trace below

![bokeh_fail_ie_08_05_2017](https://cloud.githubusercontent.com/assets/14756121/25821430/ee4a8e26-342c-11e7-9854-7a894cc5877b.PNG)

@bryevdv I have included this information to act as a possible point of reference if this issue pops up in a dev/release build.
